dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/vitals/VitalFrameCallbackTest.kt

```diff
@@ -168,8 +212,18 @@ internal class VitalFrameCallbackTest {
 
     companion object {
         const val ONE_MILLISSECOND_NS: Long = 1000L * 1000L
+        const val QUARTER_SECOND_NS: Long = 250L * 1000L * 1000L
         const val ONE_SECOND_NS: Long = 1000L * 1000L * 1000L
         const val TEN_SECOND_NS: Long = 10L * ONE_SECOND_NS
         const val ONE_MINUTE_NS: Long = 60L * ONE_SECOND_NS
+
+        val logger = InternalLoggerTestConfiguration()
```

Comment:
seems we don't need this one, because there are not tests checking `logger` invocations.